src/backend/base/langflow/components/deactivated/SubFlow.py

```diff
@@ -53,7 +53,7 @@ def update_build_config(self, build_config: dotdict, field_value: Any, field_nam
                 # Add inputs to the build config
                 build_config = self.add_inputs_to_build_config(inputs, build_config)
             except Exception as e:
-                logger.error(f"Error getting flow {field_value}: {str(e)}")
+                logger.error(f"Error getting flow {field_value}: {e}")
```

Comment:
Maybe we could use `logger.exception` here and remove the error from the message as it will be in the traceback ?

Reply:
I agree.